ons_alpha/jinja2/templates/pages/bulletins/bulletin_page.html

```diff
@@ -1,17 +1,10 @@
-{% extends "templates/base_page.html" %}
+{% extends "base_page.html" %}
 
 {% block main %}
     {{ table_of_contents(page, "body", {"title": "Table of contents"}) }}
-
     {% include_block page.updates %}
-
     {% include_block page.body %}
-
+    <div class="previous-releases">
+        <a href="{% pageurl page.get_parent.specific %}previous-releases/">Previous Releases</a>
```

Comment:
1. you don't need specific for generating URLs. This is all encapsulated by the core `Page` class in Wagtail
2. I am fairly certain Jinja requires methods to be called as such (so `page.get_parent()`)

```suggestion
        <a href="{% pageurl page.get_parent() %}previous-releases/">Previous Releases</a>
```